src/components/Blocking/ItemList.js

```diff
@@ -5,6 +5,7 @@ import Icon from 'react-native-vector-icons/FontAwesome';
 
 import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` from the theme file is a good change as it centralizes color management and improves consistency across the application. However, please ensure that the old `colors` import is no longer used in this file, otherwise it could lead to confusion or bugs.
```diff
- import {colors} from '../../utils/colors';
+ import {COLORS} from '../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
@agit26milan masih ada `import {colors} ...` better dihapus aja mas import ini karena tidak dipakai